Base/usr/share/man/man5/font.md

```diff
@@ -1,17 +1,39 @@
 ## Name
 
-font - Bitmap Font File format
+font - Bitmap Font File format (.font)
 
 ## Synopsis
 
-The .font file format stores bitmap fonts in SerenityOS's own binary format.
+Font files contain bitmap definitions of fonts (Gfx::BitmapFont).
 
 ## Description
 
-These files contain bitmap definitions of fonts, either varying-width or fixed-width.
+Bitmap fonts can be either varying-width or fixed-width.
 
-The first four bytes contain the filemagic: `!Fnt` (0x21466e74).
+There is a list of features:
+
+- The first four bytes contain the filemagic: !Fnt (0x21466e74).
+- Question mark '?' is used as a fallback for unknown glyph or emoji.
+- Gfx::BitmapFont supports read from and write to the font files (as well as read directly from the memory).
+
+## Structure
+
+| size       | member name         |
+|------------|---------------------|
+| [4 bytes]  | filemagic           |
+| [1 byte]   | glyph width         |
+| [1 byte]   | glyph height        |
+| [2 bytes]  | range mask size     |
```

Comment:
I couldn't find anything about endianness and I'm assuming it's system dependent. However, it would be nice for someone who knows more about this to take a look (I've never dealt with endianness before).

Reply:
Then can you leave an HTML comment mentioning this?

Reply:
As far as I can tell, according to BitmapFont.cpp it should be big-endian.